keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets_test.py

```diff
@@ -87,6 +92,53 @@ def test_load_weights_error(self):
                 "csp_darknet_tiny", load_weights=True
             )
 
+    @parameterized.named_parameters(
+        ("xs", "yolov8_xs_backbone_coco", "yolov8_xs_backbone"),
+        ("s", "yolov8_s_backbone_coco", "yolov8_s_backbone"),
+        ("m", "yolov8_m_backbone_coco", "yolov8_m_backbone"),
+        ("l", "yolov8_l_backbone_coco", "yolov8_l_backbone"),
+        ("xl", "yolov8_xl_backbone_coco", "yolov8_xl_backbone"),
+    )
+    def test_yolo_v8_preset_same_output(self, yolo_preset, csp_preset):
```

Comment:
It seems like this test (and the utilities built for it in e.g. `yolo_v8_backbone_presets.copy_weights` are useful now to demonstrate that the forward pass is the same, but not useful going forward as part of CI.

Would you agree?

I think it would be great if we could just get rid of the `YOLOV8Backbone` entirely, and then these tests would become obsolete.

Reply:
Yes, I totally agree.
The idea here is to keep a record of every piece of code used to produce these new weights in the master branch as a commit.
I will create a follow-up PR to remove both YOLOV8Backbone, the legacy csp code, and the related tests.